### DISCUSSION TITLE
End user opinion—com_categories needs enhancement

### DISCUSSION THREAD

**Discussion by CoalitionGuy64** [joomla/joomla-cms#28095] [2. Cold Features]
Joomla! is a fantastic CMS that is quickly becoming a personal to enterprise level publishing and distribution system. The enhancement requested here will not only streamline and simplify joomla management by end users, but will also create a higher market share / customer base.

Joomla needs a set of options in com_categories where upon creating a category, joomla will create and associate an images/media folder (possibly both) for that category as named. Having this option/set of options available will help end users in news agencies, newspapers, and other enterprise level operations simplify content management and keep site mapping much more automated and organized.

For many end users, it’s too easy to submit articles that are all over the place—causing joomla to process more files and db fields during operations. Having this option available should make things easier to create a well organized and simpler site management.

**Comment by SharkyKZ** [comment]
Some sites may want to organize media based on date (year/month), authors, sources, content items or some other criteria. Forcing some structure here is too opinionated and should be left for 3rd party extensions.

**Comment by CoalitionGuy64** [comment]
> Some sites may want to organize media based on date (year/month), authors, sources, content items or some other criteria. Forcing some structure here is too opinionated and should be left for 3rd party extensions.

That is very true.  But what I’m suggesting is an option for end users needing a more managed environment. Since the auto-creation of an image folder by category name would be optional, one could always shut it off.

Joomla[s a great CMS, but a lot of end users don’t like it because of its category and article limitations.
